src/libs/data-access/user-state/+state/user.reducer.ts

```diff
@@ -19,11 +19,18 @@ export const initialState: UserState = {
 
 export const userReducer = createReducer(
   initialState,
-  //@TODO: 6) reducer na akcje `Sign Up Success` oraz `Log In Success` ustawi otrzymane dane użytkownika i propsa isLoading na false w state'cie
   on(USER_ACTIONS.signUp, USER_ACTIONS.logIn, (state, payload) => ({
     ...state,
     isLoading: true,
   })),
+  on(USER_ACTIONS.signUp, USER_ACTIONS.signUpSuccess, (state, payload) => ({
```

Comment:
case do USER_ACTIONS.signUp jest już zaimplementowany w lini 22, i ustawia isLoading na true. Podejrzewam, że myślałeś że podawanie kilku akcji w _on_ oznacza reducer który wykona się jeśli wykonają się w takiej kolejności jak je podajemy? Jeśli podajemy kilka akcji to oznacza że zmiana stanu zadeklarowaną funkcją wykona się po każdej z nich